ui/src/app/machines/hooks.tsx

```diff
@@ -35,6 +36,7 @@ export const useToggleMenu = (
 export const useMachineActionForm = (
   actionName: MachineActionName | "check-power"
 ): {
+  errors: MachineState["eventErrors"][0]["error"][];
```

Comment:
`errors` seems like a misnomer here if it can only ever be one error

Reply:
You're right that we only return a single "error" (for one machine), but the server can return errors as a string, array or object of errors. So the "errors" could be multiple errors or just a single string...

At any rate the type here was wrong which I'll fix.